.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -27,14 +29,18 @@ jobs:
     container:
       image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.61
       volumes:
-        - /tmp:/tmp # make sure Claircore blob can access tmp folder
+        # The updater makes heavy use of /tmp files.
+        - /tmp:/tmp
     strategy:
-      fail-fast: false # If one of the versions fails, it should not stop others from succeeding.
-      max-parallel: 1 # Jobs are memory intensive, so only run one at a time.
+      # If one of the versions fails, it should not stop others from succeeding.
+      fail-fast: false
+      # The updater is memory intensive, so only run one at a time.
+      max-parallel: 1
       matrix:
-        version: ${{ fromJson(needs.read-release-versions.outputs.matrix).versions }}
+        include: ${{ fromJson(needs.read-release-versions.outputs.matrix).versions }}
     env:
       ROX_PRODUCT_VERSION: ${{ matrix.version }}
+      ROX_PRODUCT_TAG: ${{ matrix.tag }}
```

Comment:
We have an example of the outcome here: https://github.com/stackrox/stackrox/actions/runs/8273936747/job/22638534266

(Ignore each job execution, as they should purposely fail -- because I didn't want them to upload anything to the prod bucket. The matrix is what matters.)

Reply:
All good here? Can we resolve this?